### ISSUE TITLE
Notes for Archiving

### ISSUE THREAD

**Issue by Gowee** [Gowee/readability-bot#1]
The initial goal is to build a simple API and a Telegram bot that help make any page readable, relying on the [readability.js](https://github.com/mozilla/readability/blob/master/Readability.js) project by Mozilla. The service is expected to be hosted fully on Vercel, which has no storage, for no maintenance pain.

Converted pages are planned to be stored on Telegra.ph. The problem is, there is no trivial way to avoid repeated publishing reliably as the id of a published page is generated by Telegra.ph automatically, prevent us from using a hash function. Besides, it is certainly not a good idea to abuse somehow.

An alternative way is rendering readable pages on Vercel and push to readers with a custom Telegram Instant View template. But it is not trivial to write even simple UI styles.

Cloudflare Workers is also a candidate at first with KV storage support. But its runtime limit prevents running readability.js which is a little heavy. And it is a little stupid to use both Cloudflare Workers and Vercel as it makes the project harder to maintain and the repo a little messy.

To sum up, the trivial project is expected to be a trivial effort to build such a long-running service with few codes. But it turns out to be not trivial enough. 

Just archive it. There is already a working bot for this anyway: [@export_telegraph_bot](https://github.com/gaoyunzhi/telegraph_export_bot).

**Comment by Gowee** [comment]
The second way is taken. Unarchived.
